apps/aecore/test/aecore_suite_utils.erl

```diff
@@ -257,6 +264,10 @@ mine_blocks_until_txs_on_chain_loop(_Node, _TxHashes, 0, _Acc) ->
 mine_blocks_until_txs_on_chain_loop(Node, TxHashes, Max, Acc) ->
     case mine_blocks_loop(1, key) of
         {ok, [Block]} -> %% We are only observing key blocks
+          % set a properly huge key block mining interval
+          ok = rpc:call(
+              Node, application, set_env, [aecore, expected_mine_rate,
+                                           1000 * 60 * 3], 5000),
```

Comment:
180s vs. default 300s? Do you mean huge compared to rate used normally in tests?

Reply:
Although not required to be the same as real life, default is [180 s]*(https://github.com/aeternity/epoch/blob/master/apps/aecore/src/aec_governance.erl#L41-L42)